src/main/resources/Example nets/referendum-colored.tapn

```diff
@@ -198,7 +198,7 @@
       </declarations>
     </structure>
   </declaration>
-  <query active="true" algorithmOption="CERTAIN_ZERO" approximationDenominator="0" capacity="12" colorFixpoint="false" discreteInclusion="false" enableOverApproximation="false" enableUnderApproximation="false" extrapolationOption="AUTOMATIC" gcd="false" hashTableSize="MB_16" inclusionPlaces="*NONE*" name="Six voted no" overApproximation="false" pTrie="false" partitioning="false" reduction="true" reductionOption="VerifyPN" searchOption="RANDOMHEURISTIC" symmetricVars="false" symmetry="false" timeDarts="false" traceOption="NONE" type="CTL" useQueryReduction="true" useSiphonTrapAnalysis="false" useStubbornReduction="true" useTarOption="false">
```

Comment:
Why are the capacities modified?

Reply:
The 4 queries of referendum-colored and referendum-timed-colored are not satisfiable unless you put at least 9 extra tokens. There 10 voters in both case, so k-bound should be 10.